testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabGuest.java

```diff
@@ -6,14 +6,14 @@
  * A Class to facilitate writing tests for Guest Accounts
  */
 public class LabGuest {
-    private String homeUpn;
-    private String homeLabName;
-    private String homeTenantId;
-    private List<String> guestLabTenants;
+    private final String homeUpn;
+    private final String homeTenantId;
+    private final String homeDomain;
```

Comment:
What's the difference between labName and Domain? 

Also, adding a comment specifying each params in the constructor might help.

Reply:
labName `(e.g. "arlmsidlab1")` is part of the domainName `(e.g. "arlmsidlab1.us")`

Which parameter is not clear by name, I would prefer to make the name more descriptive than add comment.